pytest_mock.py

```diff
@@ -10,7 +10,7 @@
 version = '1.2'
 
 
-class MockFixture(object):
+class MockFixture:
```

Comment:
Wouldn't make this `MockFixture` an old-style class on Python 2? Is that intended?